From: Robert Luberda <robert@debian.org>
Date: Sun, 14 Feb 2016 11:44:27 +0100
Subject: Fix FTBFS on mips and mipsel

Word `mips' is a macro on mips and mipsel architectures,
so it cannot be used as a variable name there.  Rename
occurrences of `mips' variable to `nmips' to fix FTBFS.
---
 CPP/7zip/UI/Common/Bench.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/CPP/7zip/UI/Common/Bench.cpp b/CPP/7zip/UI/Common/Bench.cpp
index 7398b3f..8bc6be2 100644
--- a/CPP/7zip/UI/Common/Bench.cpp
+++ b/CPP/7zip/UI/Common/Bench.cpp
@@ -2375,18 +2375,18 @@ HRESULT Bench(
       if (start == 0)
         start = 1;
       UInt64 freq = GetFreq();
-      UInt64 mips = numMilCommands * freq / start;
+      UInt64 nmips = numMilCommands * freq / start;
       if (printCallback)
-        PrintNumber(*printCallback, mips, 5 + ((sum == 0xF1541213) ? 1 : 0));
+        PrintNumber(*printCallback, nmips, 5 + ((sum == 0xF1541213) ? 1 : 0));
 
       /*
       if (benchCallback)
-        benchCallback->AddCpuFreq(mips);
+        benchCallback->AddCpuFreq(nmips);
       */
 
       if (jj >= 3)
       {
-        SetComplexCommands(testTime, mips * 1000000, complexInCommands);
+        SetComplexCommands(testTime, nmips * 1000000, complexInCommands);
         if (jj >= 8 || start >= freq)
           break;
         // break; // change it
